src/modules/backlight.cpp

```diff
@@ -27,9 +27,26 @@ namespace modules {
       : inotify_module<backlight_module>(bar, move(name_)) {
     m_router->register_action(EVENT_DEC, [this]() { action_dec(); });
     m_router->register_action(EVENT_INC, [this]() { action_inc(); });
-
-    auto card = m_conf.get(name(), "card");
-
+    string card = m_conf.get(name(), "card");
```

Comment:
```suggestion
    auto card = m_conf.get(name(), "card", ""s);
```

We need a default value here